### PR TITLE
Add instructions to create an application outside of the /app directory

### PR DESCRIPTION
**Pull request by bercikr**
Took a few hours to figure this out and thought it could be useful to others...

### CODE REVIEW

**Comment by zamai** [comment]
Great addition to the README! 
Should be merged 👍

**Comment by tiangolo** [comment]
Thanks @bercikr for the idea!

I'm planing on modifying the image so that it uses an env var from the start, so that to override it it would just be to override that env var, without even having to modify Supervisor configs.

**Comment by zamai** [comment]
This would be even better! :)
Looking forward

On Mon, May 15, 2017 at 2:50 AM, Sebastián Ramírez <notifications@github.com

**Comment by bercikr** [comment]
👍

**Comment by tiangolo** [comment]
Thanks a lot for the idea @bercikr ! :heavy_check_mark: :bulb: 

---

It is now integrated into the latest version (with a lot of other stuff). That specific change was actually in the base image for this one.

But there's a lot of new stuff too: https://github.com/tiangolo/uwsgi-nginx-flask-docker/releases/tag/v0.3.0

---

You can now follow the instructions for a custom app directory here: https://github.com/tiangolo/uwsgi-nginx-flask-docker#custom-uwsgiini-file

Basically, to set your app to `/application`, add something like this to your `Dockerfile`:

```Dockerfile
FROM tiangolo/uwsgi-nginx-flask:python3.6

ENV UWSGI_INI /application/uwsgi.ini

ENV STATIC_PATH /application/static

COPY ./application /application
WORKDIR /application
```

:tada:
